crates/hir-expand/src/builtin/derive_macro.rs

```diff
@@ -58,8 +58,8 @@ impl BuiltinDeriveExpander {
         tt: &tt::TopSubtree,
         span: Span,
     ) -> ExpandResult<tt::TopSubtree> {
-        let span = span_with_def_site_ctxt(db, span, id);
-        self.expander()(span, tt)
+        let span = span_with_def_site_ctxt(db, span, id, Edition::CURRENT);
```

Comment:
Now I wonder, what is the edition rustc sets for these? Presumably its the edition of the standard library?